sonic_!amd64.go

```diff
@@ -0,0 +1,36 @@
+//go:build !amd64
```

Comment:
old version of go build constraint is also need, to be compatible with old go version